grunt/data/header-template-extended.txt

```diff
@@ -1,6 +1,6 @@
 /**
  * <%= package %> v<%= version %>
- *
+ * @license
```

Comment:
I actually think I want to keep this so it ends up consistently at the top of all dist files.

But then we would want to add it to the files in packages/react* so that npm-bundled files also have it.